custom/icds_reports/tasks.py

```diff
@@ -1305,6 +1305,10 @@ def collect_inactive_awws():
         'days_since_start',
         'days_inactive'
     ]
+    # removing extra columns which are not required in the report
+    extra_columns = ['no_of_days_since_start', 'no_of_days_inactive']
+    for col in extra_columns:
+        columns.remove(col)
```

Comment:
the ticket does not mention about removing columns? or is that what you are doing instead of keeping the same logic for two things that are related?

Reply:
Actually it's not mentioned on the ticket but when I discussed with the kawalpreet then came to know we don't need these columns in this report.

Reply:
Cool, the ticket was the only thing mentioned in the description, so i referred to that. Might be nice to just put things in summary if tickets are long and has lots on context, to help with easier and better reviews.